### DISCUSSION TITLE
Is there any plan to change the minimum requirement for OpenGL ES 3.0 to OpenGL ES 3.1?

### DISCUSSION THREAD

**Discussion by daemyung** [google/filament#5200] [General]
There is many benefits from OpenGL ES 3.1. For instance, the compute pipeline, the image load and store, mapping a buffer. Filament can improve the performance if leveraging OpenGL ES 3.1 features.

**Comment by romainguy** [comment]
We'd love to use ES 3.1 but we must keep 3.0 as a minimum requirement.
